pallets/loans/Cargo.toml

```diff
@@ -63,6 +64,7 @@ std = [
   "sp-io/std",
   "strum/std",
   "orml-traits/std",
+  "serde/std",
```

Comment:
Since it's optional, shouldn't we apply `?`
```suggestion
  "serde?/std",
```

Reply:
Good one!